Monika-After-Story/MonikaModDev
# Add window geometry detection/Enhanced Idle, stock

# Overhauls Windowreacts framework:
- I have refactored a lot of windowreacts to be more efficient, we're no longer running needless OS checks anymore.
- I have added API funcs to get the absolute positions of the mouse cursor and MAS window, and functions to compare the two (left of MAS window, right of MAS window, etc)
- I have also built some new changes to the sprite generator, allowing us to generate sprites where Monika will follow the mouse cursor, and implemented some into our idle sprites.

# Testing:
- Verify windowreacts still work
- Verify idle looks good
- Verify no issues in sprite generation
- [ ] Linux Testing
- [ ] Windows Testing
- [ ] Mac Testing
